body.tex

```diff
@@ -231,6 +231,43 @@ \subsection{Epic-Based Long Term Plans}
+
+\subsection{Defining the Schedule with Milestones}
+
+Rubin Milestones are defined as Jira issues of type ``Milestone''. As indicated above, the Director (or their designate) defines the L1 milestones, the Associate Directors (ADs) define their departments' L2 milestones, and the Team Leaders and ADs define the L3 milestones for their teams.
+
+The following fields must be filled out when defining a new milestone:
+\begin{itemize} 
+\item 
+  The \texttt{Milestone Level}, ``1,'' ``2,'' or ``3;''
+\item
+  The \texttt{RO Milestone ID}. 
+  This is a string like ``L2-DI-0003'' which indicates the level of the milestone, the department (for level 2 milestones) or team (for level 3 milestones) that owns it, and an ID number that serves to make the ID unique wbut which otherwise carries no meaning. 
+  The list of two-character department and team string identifiers can be found in the Appendix (``DI'' in the example stands for ``Directorate''). 
+  The ID number may get edited by the Director's Office to ensure that it is unique: the Milestone IDs are only ever used in linking together the milestones for visualization in Smartsheet. 
+  Good practice here is to look at the milestone list in Smartsheet and choose an ID number that leads to a milestone ID that has not already been taken, and which roughly fits your milestone into the time-ordered list.  
+\item
+  The \texttt{Summary} field should contain a sentence outlining either what needs to be done, or what will be delivered for the Milestone to be reached. 
+  Examples include \textit{``Deliver Data Preview 0.1 (DP0.1)''} (an L1 milestone) and \textit{``DP0.1 Data Release: science-ready catalogs released from the IDF''} (an L2 milestone that belongs to it).
+  % PJM: Should we homogenize the style here, and make Milestone summaries EITHER directives OR deliverables? Discussed with Amanda: leave this up to the teams' preferences.
+\item  
+  The \texttt{POP Activity Description} text should contain one or maybe two sentences summarizing the activity needed to complete the milestone. 
+  Example: \textit{``Upgrade the WBS activity, labor and non-labor plans from V4 to V5 in order to capture a US DF at SLAC, a UK DF, and any other modifications needed, and estimate the corresponding budget.''} 
+  Note that only L1 (and sometimes L2) milestones are actually listed in the NOIRLab Program Operations Plan (POP), but Rubin adopts the same structure for all its milestones.
+\item
+  The \texttt{POP Deliverable Description} is a very terse list of the deliverables needed to reach the Milestone. 
+  Example: \textit{``V5 WBS workbook and Preliminary Cost Calculator.''}
+\item 
+  The \texttt{Due Date} is the latest date in the future by which the milestone needs to be reached. 
+  This date should be before or the same as the milestone's parent milestone's ``Rubin Forecast Due Date'' as shown in the Smartsheet. 
+\item
+  The \texttt{Start Date} is the date when the work for the milestone should begin.
```

Comment:
At the moment we are not requiring a start date - it is assumed to be at the start of a quarter or planning period.  An issue is that Leanne's group has already start work on FY22 milestones, which haven't been formally introduced to smartsheet or had Jira issues created yet.

Reply:
I'm sure we can backfill when needed. The Jira milestone issues do have a Start Date field, which I guess we are treating as optional: if the creator doesn't enter anything, it sounds like we have a reasonable algorithm for defining a start date in Smartsheet automatically. I'll try and make this clear in the text.